quack_app/lib/main.dart

```diff
@@ -1,6 +1,12 @@
 import 'package:flutter/material.dart';
+import 'package:quack_app/core/menu_data.dart';
 
-void main() {
+import 'components/navbar.dart';
+
+void main() async {
+  MenuData menuData = MenuData();
```

Comment:
It says it's not covered by tests. Is there a way to add tests for this?

Reply:
Nice job catching that! :D I will look into it. `MenuData` has its own tests in `core`. I am not sure how we would test the MenuData creation here. Any ideas?

Reply:
Hmm yeah and I did look at the other tests you made, I don't think I'm familiar enough with testing yet with our stuff to offer much help tbh. I think once I start writing/running the code I would understand it more and have more ideas

Reply:
There doesn't seem to be a way to test this :(